app/javascript/controllers/editor_controller.js

```diff
@@ -0,0 +1,35 @@
+import { Controller } from "@hotwired/stimulus"
+import Prism from 'prismjs';
+import '../prism-languages';
+import codeSyntaxHighlight from '@toast-ui/editor-plugin-code-syntax-highlight';
+import Viewer from '@toast-ui/editor/dist/toastui-editor-viewer';
+import Editor from '@toast-ui/editor';
+
+export default class extends Controller {
+  static targets = ["field"]
+
+  onChange() {
+    this.fieldTarget.value = this.editor.getMarkdown();
+  }
+
+  connect() {
+    this.editor = new Editor({
+      el: document.getElementById('editor'),
+      previewStyle: 'vertical',
+      height: '500px',
+      initialEditType: 'markdown',
+      initialValue: '```ruby\ndef something\n  return true\nend\n```',
```

Comment:
Do we want this? Maybe is better to leave it blank.

Reply:
We dont! I just changed it to pull the intended initialValue on render. It should display nothing for a new post and the actual post markdown content for the edit view.